src/concrete/ml/sklearn/base.py

```diff
@@ -1859,6 +1860,14 @@ def fit(self, X: Data, y: Target, **fit_parameters):
 
         self._is_fitted = True
 
+        # To enable the AutoRounder feature, it has to be adjusted using an inputset to determine
+        # how many of the least significant bits to remove.
+        # This can be done manually using fhe.AutoRounder.adjust(function, inputset), or
+        # by setting auto_adjust_rounders configuration to True during compilation.Rounding
+        # We chose to adjust it manually to get the same result between clear and FHE inference
+        inputset = numpy.array(list(numpy.expand_dims(q_input, 0) for q_input in self._q_fit_X))
```

Comment:
may be better to use the `_get_inputset_generator` utils from quantized modules (I think it does exactly that but probably check before)